### DISCUSSION TITLE
My industry session shows incomplete

### DISCUSSION THREAD

**Discussion by DeveloperRVerma** [github/india#475] [General]
I have watched 4 industry sessions but it shows incomplete. I am so helpless right now please let me know what should I do now. After watching the videos it shows complete for a short period of time but when I log in again the next day it shows incomplete except the one video.

Please help me out of this situation.
[Learning _ Future Ready Talent.pdf](https://github.com/github/india/files/7389219/Learning._.Future.Ready.Talent.pdf)

[Learning _ Future Ready Talent.pdf](https://github.com/github/india/files/7389219/Learning._.Future.Ready.Talent.pdf)

**Comment by aparidhi9** [comment]
Yes, same issue I am facing and even I am not able to access to self learning courses. 
Please help me out:)
